### ISSUE TITLE
Advanced GBT mempool crash on block getting mined

### ISSUE THREAD

**Issue by wiz** [mempool/mempool#3029]
Running 241850b in prod crashed just now:
```
Feb 10 10:42:37 [46939] DEBUG: New block found (#775870)!
Feb 10 10:42:38 [46939] DEBUG: Indexing tx 1 of 3032 in block #775870
Feb 10 10:42:38 [46939] DEBUG: 3031 of 3032 found in mempool. 1 fetched through backend service.
/mempool/mainnet/backend/dist/api/websocket-handler.js:416
        const auditMempool = (config_1.default.MEMPOOL.ADVANCED_GBT_AUDIT === config_1.default.MEMPOOL.ADVANCED_GB
T_MEMPOOL) ? _memPool : JSON.parse(JSON.stringify(_memPool));
                                                                                                                  
                                        ^

RangeError: Invalid string length
    at JSON.stringify (<anonymous>)
    at WebsocketHandler.handleNewBlock (/mempool/mainnet/backend/dist/api/websocket-handler.js:416:155)
    at /mempool/mainnet/backend/dist/api/blocks.js:467:78
    at Array.map (<anonymous>)
    at Blocks.$updateBlocks (/mempool/mainnet/backend/dist/api/blocks.js:467:66)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Server.runMainUpdateLoop (/mempool/mainnet/backend/dist/index.js:182:13)


```

**Comment by nymkappa** [comment]
Maybe related? https://stackoverflow.com/questions/29175877/json-stringify-throws-rangeerror-invalid-string-length-for-huge-objects

**Comment by nymkappa** [comment]
Fixed in https://github.com/mempool/mempool/pull/3030 ?

**Comment by wiz** [comment]
Fixed in #3030 !
